core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -239,6 +244,13 @@ private void removeMetrics() {
         remoteStorageReaderThreadPool.removeMetrics();
     }
 
+    /**
+     * Returns the timeout for the RLM Tasks to wait for the quota to be available
+     */
+    Duration quotaTimeout() {
+        return Duration.ofSeconds(1);
```

Comment:
Can you use the declared constant value?

Reply:
I want to override this method to have a smaller timeout for tests. That is the reason I did not use a constant value.